javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/TracerInstaller.java

```diff
@@ -70,6 +70,30 @@ private static synchronized void installExporters(String exporterName) {
       log.warn("No {} span exporter found", exporterName);
       log.warn("No valid span exporter found. Tracing will run but spans are dropped");
     }
+
+    MetricExporterFactory metricExporterFactory = findMetricExporterFactory(exporterName);
+    if (metricExporterFactory != null) {
+      DefaultExporterConfig config = new DefaultExporterConfig("exporter");
+      installExporter(metricExporterFactory, config);
+    } else {
+      log.warn("No {} metric exporter found", exporterName);
```

Comment:
will this log warning for zipkin users?

Reply:
good point! Yes it will.  I can remove this warning to reduce the confusion to jaeger/zipkin users if you don't mind.

Reply:
Sounds good, maybe `debug` then?

Reply:
yep, just updated.